sdk/servicebus/service-bus/src/core/linkEntity.ts

```diff
@@ -180,7 +180,7 @@ export abstract class LinkEntity<LinkT extends Receiver | AwaitableSender | Requ
    * @param options Options that can be provided while creating the LinkEntity.
    */
   constructor(
-    name: string,
+    public entityPath: string,
```

Comment:
Well... In case of senders and receivers, this indeed will be the entityPath, but in case of the management link, this would be the entityPath followed by `$management`.

We could consider keeping this parameter as `name` and expect the classes that extend LinkEntity to use the `getUniqueName()` method and pass the unique name.

Reply:
Let me check - the only reason I exposed this was to allow getManagementClient(<entitypath>) to work in the receiver. I'll trace the value a bit more.

Reply:
I changed this to be two parameters (basically) - 'baseName' which is used when generating the link name and 'entityPath' which is the actual entity Path. 

This at least allows the actual parameter and value to be more clear.

Reply:
(I haven't changed the field we use for the key in this PR. We might want to consider that next after some Q&A with the service team)